roles/vdm/tasks/postgres/postgres.yaml

```diff
@@ -111,7 +110,7 @@
         cadence_number: "{{ V4_CFG_CADENCE_VERSION }}"
         existing: "{{ vdm_overlays }}"
         add:
-          - { transformers: "overlays/external-postgres/external-postgres-transformer.yaml" }
+          - { resources: "overlays/postgres/platform-postgres", min: "2022.10" }
```

Comment:
What if I have an older order, what and how does the external-postgres-transformer.yaml get put in place as the v5 operator will not be in the mix?